src/service/command-executor.js

```diff
@@ -44,6 +47,36 @@ function testCopyJUnitReports(outputDir) {
   files.forEach((file) => fs.copyFileSync(path.join(sampleDir, file), path.join(outputDir, file)));
 }
 
+function checkFileExist(ksProjectPath, newTestSuitePath) {
```

Comment:
This function should be moved to a util. Moreover, this should be a generally-applicable function, not just for test suite path.

Reply:
Update move this function to util and rename params.